### ISSUE TITLE
Step 6: Issue Page Clarification

### ISSUE THREAD

**Issue by sauravghosal** [open-learning-exchange/open-learning-exchange.github.io#2793]
### Problem
Confusion with what's going on in Step 6: Issues. I'm quite confused about how the pull request/branching plays into creating an issue, and I think having a step-by-step solution or an example going through creating one issue would be very helpful. 

### Steps to reproduce the problem
N/A

### Screenshots
![issues page](https://user-images.githubusercontent.com/33039426/67151083-a3dda300-f28e-11e9-81af-61791513ffc2.PNG)

### Proposed solution
- Providing a step-by-step example going through creating a new issue - *maybe a video tutorial*? I don't learn very well by passively reading text, and I find videos more helpful.

**Comment by jholtkort** [comment]
@sauravghosal I can see what you mean.  Perhaps a summary above the "Create an Issue" section would be helpful (such as this one from the 'Git Repositories' tutorial from step 5:

![Screen Shot 2019-10-23 at 1 56 02 PM](https://user-images.githubusercontent.com/47533712/67425044-e6f48a80-f59c-11e9-84f4-bf9320d1ac83.png)

**Comment by sauravghosal** [comment]
Yeah, I think that could help! Only after creating my first issue did I really learn how it all worked. If there was an concrete example walkthrough in the tutorial, I would've found that very helpful as well.

**Comment by johnbeatrice** [comment]
More resources and information could be helpful with learning the workflow. Maybe there are solid videos already out there that teach this? and they could just be linked for additional reference.

**Comment by Shi-web** [comment]
I believe there is already a link which breaks down the pull request/ branching quite well for step 6. This issue is already solved, I think it is a better idea to close this issue as it has been already solved!
